store/policy.go

```diff
@@ -323,7 +323,68 @@ func (s *Store) getPolicyRaw(ctx context.Context, find *api.PolicyFind) (*policy
 	return ret, nil
 }
 
-func findPolicyImpl(ctx context.Context, tx *Tx, find *api.PolicyFind) ([]*policyRaw, error) {
+func findPolicyImpl(ctx context.Context, tx *Tx, find *api.PolicyFind, mode common.ReleaseMode) ([]*policyRaw, error) {
+	if mode == common.ReleaseModeDev {
+		// TODO(d): handle resource type.
+		// Build WHERE clause.
+		where, args := []string{"1 = 1"}, []interface{}{}
+		if v := find.ID; v != nil {
+			where, args = append(where, fmt.Sprintf("id = $%d", len(args)+1)), append(args, *v)
+		}
+		if v := find.EnvironmentID; v != nil {
+			where, args = append(where, fmt.Sprintf("resource_id = $%d", len(args)+1)), append(args, *v)
```

Comment:
Let's add a comment to this field definition, otherwise, other people could get confused whether it's a typo

Reply:
Done. Updated the field name entirely. Was trying to do so for the next PR.

Reply:
The current find implementation won't use the (resourceType, resourceID) index

So we either add ResourceType = PolicyResourceTypeEnvironment here or pass it from PolicyFind?

Reply:
Done.

Was trying to do so for the next PR with TODO above. We don't have a lot of policies so I'm not worried too much about indices.